hot/log.js

```diff
@@ -0,0 +1,14 @@
+var logLevel = "info";
+
+module.exports = function(level, msg) {
+	if(logLevel === "info" && level === "info")
```

Comment:
Actually, I also wanted to refactor it, but I thought one change at a time is better, so I just copy/pasted it, as I said. You can check [this generic solution](https://gist.github.com/lbogdan/75be2d412d8e6c9d64a909e1d019fbe6).